frontend/public/components/utils/details-item.tsx

```diff
@@ -94,9 +95,12 @@ export const DetailsItem: React.FC<DetailsItemProps> = ({
                 {...(path && { footerContent: <PropertyPath kind={model?.kind} path={path} /> })}
                 maxWidth="30rem"
               >
-                <Button data-test={label} variant="plain" className="details-item__popover-button">
-                  {label}
-                </Button>
+                <Button
+                  icon={label}
```

Comment:
The details-item `{label}` text is now being passed in to the icon prop. This change looks to be part of a codemod run.
 
![Screenshot 2025-01-17 at 6 47 29 PM](https://github.com/user-attachments/assets/c0608ef0-48e8-4fa5-af64-53044aa4d597)

Reply:
That's definitely a mistake.  No `icon` prop should be passed there.